src/css/block.header.css

```diff
@@ -0,0 +1,38 @@
+/*------------------------------------*\
+    # block.header
+\*------------------------------------*/
+
+.header {
+    position: relative;
+    background-color: $color-primary;
+    padding: 0 calc($unit * 3);
+    align-items: center;
+    overflow: hidden;
+    box-shadow: 0 4.5px 4px rgba(0, 0, 0, 0.035),
+        0 12.5px 11px rgba(0, 0, 0, 0.05), 0 30.1px 26.5px rgba(0, 0, 0, 0.065),
+        0 100px 88px rgba(0, 0, 0, 0.1);
+}
+
+.wrapper {
+    display: flex;
+    align-items: center;
+    padding: calc($unit * 5) 0;
+}
+
+.header--button-wrapper {
+    position: absolute;
+    right: 0;
+}
+
+.title {
+    cursor: default;
+    margin: auto;
+}
+
+.title--header-white {
+    color: $color-light;
```

Comment:
Postoji još jedan tip CSS-a koji pišemo i koristimo, a zovu se util klase.

To su mikro klase koje setaju jedan property i mogu se svagdje koristiti.

Tipa, možeš napisati util: `.t-center {text-align: center}` ili `t-primary {color: $primary;}`. "t" je u ovom slučaju skraćeno "typography".

Prouči css fileove sa "util"prefiksom: https://bitbucket.org/prototypdigital/investors-club/src/develop/src/css/

Vidi što možeš zamjeniti sa utilima i da ti klase budu još manje.